### ISSUE TITLE
$_SESSION

### ISSUE THREAD

**Issue by kmonmousseau** [TheNohman/GuidapLuckyNumberTest#4]
Symfony te permet de récupérer les données de session sans utiliser la variable globale, et c'est plus dans l'esprit du framework.

**Comment by TheNohman** [comment]
Oui, fallait juste que je retrouve comment faire via le framework, 
Du coup, le changement est dans ApiNumberController maintenant.
